### ISSUE TITLE
[Feature Request] Include $value in column closure

### ISSUE THREAD

**Issue by tabacitu** [Laravel-Backpack/community-forum#73]
# Feature Request

### What's the feature you think Backpack should have?

TLDR: I wish there was a `$value` variable inside column closures. Maybe even field, filter and widget, haven't thought if that applies to them too.

Here's how I usually use the `wrapper` functionality:
```php
        CRUD::column('type')->wrapper([
            'element' => 'span',
            'class' => function ($crud, $column, $entry) {
                switch ($entry->type) {
                    case 'production':
                    case 'prod':
                        return 'badge badge-pill badge-success';
                        break;

                    case 'staging':
                    case 'testing':
                        return 'badge badge-pill badge-warning';
                        break;
                    
                    case 'development':
                    case 'local':
                        return 'badge badge-pill badge-default';
                        break;

                    default:
                        return 'badge badge-pill badge-danger';
                        break;
                }
            }
        ]);
```

I know it's small, but every time, I have to think about what column that's on, what's the column name. And when I copy-paste this `wrapper` array to another column, I have to change it.

But... most of the time, what you want is actually `$value`. That's what you're using for your conditionals. So it would be a little easier if we could do:

```diff
        CRUD::column('type')->wrapper([
            'element' => 'span',
            'class' => function ($crud, $column, $entry) {
-                switch ($entry->type) {
+                switch ($value) {
                    case 'production':
                    case 'prod':
                        return 'badge badge-pill badge-success';
                        break;

                    case 'staging':
                    case 'testing':
                        return 'badge badge-pill badge-warning';
                        break;
                    
                    case 'development':
                    case 'local':
                        return 'badge badge-pill badge-default';
                        break;

                    default:
                        return 'badge badge-pill badge-danger';
                        break;
                }
            }
        ]);
```

It's easier to read. And easier to understand. You're making a conditional based on that column's value, for that entry. And it's not terribly difficult to achieve either, we just have to do `$value = $entry->{$column['name']};` or `$value = data_get($entry, $column['name']);` somewhere, so that `$value` is available inside the closure. Of course, we'd also need to treat the case where that column does not exist, neither in the DB nor as an accessor or model function.

### Have you already implemented a prototype solution, for your own project?

Not yet!

### Do you see this as a core feature or an add-on?

Core.
